### ISSUE TITLE
Error when searching for a mobile phone

### ISSUE THREAD

**Issue by rosmander** [TheNetsky/Microsoft-Rewards-Script#123]
![Sin título](https://github.com/TheNetsky/Microsoft-Rewards-Script/assets/58051240/dab2259e-e933-4677-806c-38f1295a2ec8)
So far everything has been working perfectly, but since two days ago the script does not perform mobile searches, giving me the bad-user agent error, I have updated playwright and I have even installed all its modes "npm init playwright@latest" but the error persists, do you know what could be the cause of the error?

**Comment by rosmander** [comment]
hi?

**Comment by TheNetsky** [comment]
I cannot replicate this.

**Comment by rosmander** [comment]
Ok, thanks

**Comment by lscheub** [comment]
This also happens for me on one of the accounts. That specific account has a warning at the top of the rewards site "Unusual search activity might be influencing your ability to earn points" (translated from german). This is at least for me why it does not gain any mobile search points and consequently retries the mobile searches endlessly.

**Comment by rosmander** [comment]
I have not looked at this warning the truth, I will do it soon, what is true is that if the script gives me error and I use my mobile phone, the error disappears, hence I thought it was some problem of the user agent generated by the script and I asked here.

**Comment by xunilalala** [comment]
I have encountered this issue before, and my solution is as follows.

Initially, everything worked normally, but over time, the mobile search stopped working properly while the desktop search continued to function. I preliminarily deduced that it was a session issue.

Over the following month, by using my solution, the issue did not reoccur.

My solution: Clear the session every day at midnight and run this script three times daily.

**Comment by xunilalala** [comment]
> This also happens for me on one of the accounts. That specific account has a warning at the top of the rewards site "Unusual search activity might be influencing your ability to earn points" (translated from german). This is at least for me why it does not gain any mobile search points and consequently retries the mobile searches endlessly.

Please search the previous issue records. I remember there was one mentioning that certain regions have time restrictions on search, possibly requiring a minimum of 15 minutes between every three searches. Please try to find it.

**Comment by rosmander** [comment]
> Please search the previous issue records. I remember there was one mentioning that certain regions have time restrictions on search, possibly requiring a minimum of 15 minutes between every three searches. Please try to find it.

Yes, this is a problem that microsoft randomly assigns to some accounts for a while and then disappears. That happens to me in one of my accounts, and what I did was to ask a programmer friend to add to the script a 15 minute counter every 4 searches and that's it, it still gives me the error, but because I did not know what you have indicated above to delete the data.

Thank you very much for your attention and answers.

**Comment by xunilalala** [comment]
@rosmander 
Bro,

First, it's a pity that you had to ask a programmer friend to help you with the code because the main repository fully supports this feature. You just need to adjust the values in the config file.
![image](https://github.com/TheNetsky/Microsoft-Rewards-Script/assets/64685627/4f2c6d84-771d-424c-a729-1ac854a34da3)

Second, if you only see the cmd window, it might be because "headless" is set to true in the config or you are using Docker.

Third, the session files are located in ./browser/sessions. Try deleting the sessions folder.

**Comment by rosmander** [comment]
> ./browser/sessions

Good again, thank you very much for telling me the command that would have to be put to pause between searches, however, after several days, yesterday microsoft removed the limitation of 3 searches every 15 minutes of that account.

My friend also had to modify the login file, because I use Authenticator not using a password and if I did it as the script said it would give me problems, now it does not.

If I have the headless option on and do not use docker, with the modification that she made me I get in the cmd the code that I have to enter in the phone and still no problems, the truth is that this is very hard work.

Thank you very much for helping me, I have deleted today the session file and I am trying to see if it eliminates the errors as you say that gave me lately the script. At the moment it is working great, thank you very much for your help from the bottom of my heart.
